src/models/classes/factoryorder.js

```diff
@@ -0,0 +1,10 @@
+class FactoryOrder extends OrderCore
+{
+    constructor(products)
+    {
+        var time = GAME.model.config.orderTransportDuration;
```

Comment:
Time will be implemented when the 'demand' is ready I belief. Time that is left before the cusomter will leave.